docs: updated bridge migration guide (#8471)
nuxt/framework: docs/content/6.bridge/1.overview.md

})
```

::alert

> This `useHead` composable uses `@vueuse/head` under the hood (rather than `vue-meta`) to manipulate your `<head>`.  You need to add `@vueuse/head` to your package.json file for it to work properly.
> 
> ::alert{type=warning}
> We recommend not to use both the native Nuxt 2 `head()` properties as well as `useHead`, as they may conflict.

Reviewer: ```suggestion
This `useHead` composable uses `@vueuse/head` under the hood (rather than `vue-meta`) to manipulate your `<head>`. You need to add `@vueuse/head` to your package.json file for it to work properly.
::

::alert{type=warning}
We recommend not using the native Nuxt 2 `head()` properties in addition to `useHead`, as they may conflict.
::
```
